style-guide.md

```diff
@@ -123,28 +123,15 @@ Readers can often infer when a "which" clause is restrictive versus unrestrictiv
 
 ### Subject and Pronoun Disagreement
 
-Ensure that pronouns and possessives match the subject they are referring to.
+Singular "they" is acceptable as a pronoun. Make sure that there is no possibility of confusion about to what the pronoun refers (subject vs. object).
```

Comment:
```suggestion
Singular "they" is accepted as a pronoun. Make sure that there is no possibility of confusion about to what the pronoun refers (subject vs. object).
```

Reply:
imo sounds better but not too important

Reply:
Just changed to be active instead of passive, as I should  have done in the first place.

Reply:
Yep! Sounds really good now.